livre/core/Maths/Maths.cpp

```diff
@@ -61,26 +60,52 @@ void getRotationAndEyePositionFromModelView( const Matrix4f& modelviewMatrix,
     iMv.get_translation( eye );
 }
 
-Matrix4f computeModelViewMatrix( const Matrix3f& rotationMatrix, const Vector3f& eye )
-{
-    Matrix4f rotationTranspose = Matrix4f::IDENTITY;
-    rotationTranspose.set_sub_matrix( rotationMatrix, 0, 0 );
-    rotationTranspose = transpose( rotationTranspose );
-    Matrix4f modelviewMatrix = Matrix4f::IDENTITY;
-    modelviewMatrix.set_translation( -eye );
-    return rotationTranspose * modelviewMatrix;
-}
-
 Matrix4f computeModelViewMatrix( const Matrix4f& rotationMatrix, const Vector3f& eye )
```

Comment:
rotationMatrix here is Matrix4f and it looks a bit fishy ( who did that I dont know ;) ). It should better be Matrix3f.